tasks/interactive-bots.md

```diff
@@ -168,14 +180,15 @@ messages to users.
                                          'from %s:' % (original_sender,))
   ```
 
+
 * Create a links bot in settings for an existing user. Use its credentials in
 `~/.zuliprc-local`. Subscribe the bot to `devel` and `social` streams.
 
 * Check that the bot is working as expected:
 
   - it reacts to the `@link` and `@resource` at the beginning of a message
 
-  - it currently posts the message to the followup stream, as the followup bot
+  - it currently posts the message to the links stream, as the followup bot
```

Comment:
@showell @femnad here's the current dilemma. The part of the task that is kind of confusing is the part which deals with the infinite loop (`if['display_recipient'] == 'followup`). However, in reality, if the user copied the code from `followup.py`, it should post in the followup stream (I think I made a typo here).